core/scene.go

```diff
@@ -124,11 +124,11 @@ const (
 	// copy the image up to the GPU or not.
 	sceneImageUpdated
 
-	// scenePrefSizing means that this scene is currently doing a
+	// sceneContentsSizing means that this scene is currently doing a
 	// PrefSize computation to compute the size of the scene
 	// (for sizing window for example); affects layout size computation
 	// only for Over
-	scenePrefSizing
+	sceneContentsSizing
```

Comment:
`sceneSizingToContent`?

Reply:
nope, see above.

Reply:
`sceneContentSizing` implies that it is sizing the content, which is not true.

Reply:
that is precisely what it is doing.